src/crypto/crypto_context.cc

```diff
@@ -812,6 +815,58 @@ void GetRootCertificates(const FunctionCallbackInfo<Value>& args) {
       Array::New(env->isolate(), result, arraysize(root_certs)));
 }
 
+MaybeLocal<Array> X509sToArrayOfStrings(Environment* env,
+                                        const std::vector<X509*>& certs) {
+  ClearErrorOnReturn clear_error_on_return;
+  v8::EscapableHandleScope scope(env->isolate());
```

Comment:
nit: add `v8::EscapableHandleScope` to the `using` list at the top of the namespace.